thirdparty/zip/miniz.h

```diff
@@ -4401,7 +4401,7 @@ static MZ_FORCEINLINE mz_bool mz_zip_array_push_back(mz_zip_archive *pZip,
                                                      mz_zip_array *pArray,
                                                      const void *pElements,
                                                      size_t n) {
-  if (0 == n)
```

Comment:
Hi @JalonSolov! Yea, maybe this will create confusion for some. But as for me, you can make the assignments as follows (if necessary):
```c
if ((n = 0) == 0)
;
```

Reply:
Certainly.  This technique was simply to avoid errors in the code (thinking you typed `==` when you only typed `=`) that would only be noticed at runtime, instead of the compiler catching it and telling you about it.